GameDataParser/Parsers/AnimationParser.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections.Generic;
+using System.Xml;
+using GameDataParser.Crypto.Common;
+using GameDataParser.Files;
+using Maple2Storage.Types.Metadata;
+
+namespace GameDataParser.Parsers
+{
+    public class AnimationParser : Exporter<List<AnimationMetadata>>
+    {
+        public AnimationParser(MetadataResources resources) : base(resources, "animation") { }
+
+        protected override List<AnimationMetadata> Parse()
+        {
+            // Iterate over preset objects to later reference while iterating over exported maps
```

Comment:
could remove this comment since it makes no sense :P

Reply:
yea, was from a copy paste :D